src/v/config/configuration.cc

```diff
@@ -4078,6 +4078,15 @@ configuration::configuration()
       {.needs_restart = needs_restart::yes, .visibility = visibility::tunable},
       30s,
       {.min = 1s, .max = 60s})
+  , datalake_translator_flush_bytes(
+      *this,
+      "datalake_translator_flush_bytes",
+      "Size, in bytes, of the amount of per translator data that may be "
+      "flushed to disk before the translator will upload and remove its "
+      "current on disk data.",
+      {.needs_restart = needs_restart::yes, .visibility = visibility::tunable},
+      32_MiB,
+      {.min = 1_MiB})
```

Comment:
it's hard to say if there is a meaningful maximum, and besides, its nice for testing to be able to crank this up.